components/03-modules/active-filters/_active-filters.scss

```diff
@@ -0,0 +1,74 @@
+$active-filters__close-size          : 48px !default;
+$active-filters__remove-padding      : 0 !default;
+$active-filters__icon-fill           : $gray !default;
+$active-filters__icon-size           : $font-size-base !default;
+$active-filters__label-font-weight   : $font-weight-bold !default;
+$active-filters__clear-border        : 2px solid $red !default;
+$active-filters__clear-height        : 48px !default;
+$active-filters__clear-font-size     : 14px !default;
+$active-filters__clear-color         : $red !default;
+$active-filters__clear-font-weight   : $font-weight-bold !default;
+$active-filters__clear-color\@medium : $color-secondary !default;
+
+.active-filters {
+    &__item {
+        display: flex;
+        align-items: center;
+    }
+
+    &__remove {
+        width: $active-filters__close-size;
+        height: $active-filters__close-size;
+        padding: $active-filters__remove-padding;
+        border: none;
+        outline: none;
+        background: none;
+        cursor: pointer;
```

Comment:
Optional: I think for border and background we could add variables as I saw in some projects before where this elements had styled border / background so it can be useful.

Reply:
What do you think about changing close-size to "remove-size" or  remove-padding to "close-size". I see it's not used any other place and it's little inconsistent now

Reply:
Done.